src/components/StatusUpdater/UpdateStatusForm.tsx

```diff
@@ -131,6 +131,7 @@ export const UpdateStatusForm = ({ form }: { form: FormLogic }) => {
 
       // Set fields with dynamic default values.
       form.setValue('miles', participant?.miles ?? '');
+      form.setValue('statusTime', new Date().getTime());
```

Comment:
Sanity checking, this value should be what you have here (new Date().getTime()) rather than what was there previously (formatDate(currentTime, TextBoxDateFormat))?

I'm guessing the different value is a result of all the changes to make everything consistent, but want to make sure.

Reply:
That's correct; the old version was passing a string value whereas the new value takes the straight time value.

Reply:
Sounds good! PR looks good to me, then.